deepfence_frontend/apps/dashboard/src/features/vulnerabilities/pages/RuntimeBom.tsx

```diff
@@ -180,9 +180,16 @@ const RuntimeBom = () => {
           );
         },
         header: () => 'Type',
-        minSize: 200,
-        size: 300,
-        maxSize: 500,
+        minSize: 50,
+        size: 100,
+        maxSize: 200,
+      }),
+      columnHelper.accessor('node_name', {
+        cell: (cell) => cell.getValue(),
+        header: () => 'Name',
+        minSize: 100,
+        size: 200,
+        maxSize: 300,
       }),
       columnHelper.accessor('node_id', {
```

Comment:
we need to remove node_id column and just use node_name column

Reply:
node_id is not very useful here

Reply:
removed the node_id and updated with node_name